src/main/java/org/jitsi/videobridge/cc/BitrateController.java

```diff
@@ -945,6 +954,10 @@ private TrackBitrateAllocation(
             int preferredIdx = 0;
             for (RTPEncodingDesc encoding : encodings)
             {
+                if (encoding.getHeight() > this.maxFrameHeight)
```

Comment:
does this work ok with portrait resolutions? an iPhone would send 720x1280, which would not pass the 720 test, but I have no idea if the getHeight method accommodates for this?